prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -616,3 +616,31 @@ periodics:
           requests:
             memory: 200Mi
             cpu: 80m
+
+- name: kyma-components-version
+  decorate: true
+  cron: "0 4 * * 1" # At 04:00 on every Monday
+  labels:
+    preset-stability-checker-slack-notifications: "true"
+  extra_refs:
+    - <<: *test_infra_ref
+      base_ref: master
+    - <<: *kyma_ref
+      base_ref: master
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1
```

Comment:
we should use images from here: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang

Please check if eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1 is used in any other place

Reply:
`eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1` is used in follow periodic jobs:
- orphaned-disks-cleaner
- orphaned-clusters-cleaner
- orphaned-loadbalancer-cleaner